### DISCUSSION TITLE
Change boostrap class

### DISCUSSION THREAD

**Discussion by mdilauro39** [rappasoft/laravel-livewire-tables#1247] [Q&A]
I would like to change the Bootstrap style of the filters in the following code:

 public function filters(): array
    {
        return [
            DateFilter::make('Desde')->filter(function (Builder $builder, string $value) {
                $builder->where('fecha_situacion', '>=', $value)
                ->setFilterSlidedownColspan('2');
            }),
            DateFilter::make('Hasta')->filter(function (Builder $builder, string $value) {
                $builder->where('fecha_situacion', '<=', $value);
            })
        ];
    }

**Comment by lrljoe** [comment]
You can publish the views and make adjustments there for now.

Or you may want to just do some targeted overrides in your CSS instead.

**Comment by lrljoe** [comment]
I'm looking for v3.x at exposing methods for this.

**Comment by lrljoe** [comment]
So more comprehensive access to classes is a 3.1+ most likely as I don't want to keep delaying 3.0

**Comment by lrljoe** [comment]
So FilterLabel now has a method to adjust the attributes.

Filters themselves is on the list, along with custom view per filter instance.  I'd expect this mid November at the absolute latest, as I am reviewing Filters at the moment.
